code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -98,13 +105,28 @@
 				update_appearance()
 		else
 			to_chat(user, span_warning("Seems like someone already drew something on this helmet's visor!"))
+	if(isclothing(hitting_item))
+		var/obj/item/clothing/hitting_clothing = hitting_item
+		if(hitting_clothing.clothing_flags & PLASMAMAN_HELMET_EXEMPT)
+			to_chat(user, span_notice("You cannot place [hitting_clothing.name] on helmet!"))
+			return
+		if(istype(hitting_clothing, /obj/item/clothing/head))
```

Comment:
check this first instead of checking if its clothing first, You cant do anything with it if its only clothing and not a hat anyway